packages/directlinespeech/__tests__/utilities/fetchSpeechCredentialsWithCache.js

```diff
@@ -11,6 +11,18 @@ async function fromWaterBottle() {
 
   const { region, token: authorizationToken } = await res.json();
 
+  if (enableInternalHTTPSupport) {
+    const directLineTokenResult = await fetch('https://webchat-waterbottle.azurewebsites.net/token/directline');
+
+    if (!directLineTokenResult.ok) {
+      throw new Error(`Failed to fetch Cognitive Services Direct Line credentials, server returned ${directLineTokenResult.status}`);
+    }
+
+    const { token: directLineToken } = await directLineTokenResult.json();
+
+    return { authorizationToken, region, directLineToken };
```

Comment:
@rohit21agrawal Can you fix the sorting issue here? 

```suggestion
    return { authorizationToken, directLineToken, region };
```